src/locales/en/fields.yml

```diff
@@ -181,6 +181,19 @@ selfAssessmentPaymentDate:
   legend: Date of payment
   hint: For example 31 3 2024
   content: ""
+  validation:
+    default: "Date of payment must be a real date"
+    required: "Enter your Date of payment"
```

Comment:
I think we expected that it would do the validation per field, so it would be eg "Date of payment must include a day between 1 and 31" if day/month/year were all missing - does this work if you remove `required` ?